src/tree/configurations/IngressItem.ts

```diff
@@ -7,17 +7,20 @@ import { ContainerApp, Ingress } from "@azure/arm-appcontainers";
 import { createGenericElement } from "@microsoft/vscode-azext-utils";
 import { AzureSubscription, ViewPropertiesModel } from "@microsoft/vscode-azureresources-api";
 import { ThemeIcon, TreeItem, TreeItemCollapsibleState } from "vscode";
-import { IngressConstants, azResourceContextValue } from "../../constants";
+import { IngressConstants } from "../../constants";
 import { localize } from "../../utils/localize";
 import { treeUtils } from "../../utils/treeUtils";
 import { ContainerAppModel } from "../ContainerAppItem";
 import { ContainerAppsItem, TreeElementBase } from "../ContainerAppsBranchDataProvider";
 
 const label: string = localize('ingress', 'Ingress');
 
-export class IngressItem implements ContainerAppsItem {
-    static contextValue: string = 'ingress|enabled';
-    readonly contextValue: string = `${IngressItem.contextValue}|${azResourceContextValue}`;
+export const targetPortItemContextValue: string = 'targetPortItem';
+export const visibilityItemContextValue: string = 'visibilityItem';
```

Comment:
I've been exporting the `item`-based context values to convey reusability even though some are only used in the one containing file.  I'm totally cool with removing the exports if people prefer that.

Reply:
I'd export only what is used outside the file